wic/omnect-os.rpi.wks.inc

```diff
@@ -1,5 +1,5 @@
-part --fixed-size ${OMNECT_PART_SIZE_BOOT}K      --fstype=vfat --align 4096 --label boot   --source bootimg-partition --active
-part --fixed-size ${OMNECT_PART_SIZE_UBOOT_ENV}K --no-table --offset ${OMNECT_PART_OFFSET_UBOOT_ENV1} --source rawcopy --sourceparams="file=omnect_uboot_env_ff.img"
-part --fixed-size ${OMNECT_PART_SIZE_UBOOT_ENV}K --no-table --offset ${OMNECT_PART_OFFSET_UBOOT_ENV2} --source rawcopy --sourceparams="file=omnect_uboot_env_ff.img"
+part --no-table --offset ${OMNECT_PART_OFFSET_UBOOT_ENV1} --source rawcopy --sourceparams="file=uboot-env.bin"
+part --no-table --offset ${OMNECT_PART_OFFSET_UBOOT_ENV2} --source rawcopy --sourceparams="file=uboot-env.bin"
+part --offset ${OMNECT_PART_OFFSET_BOOT} --fixed-size ${OMNECT_PART_SIZE_BOOT}K --fstype=vfat --label boot --source bootimg-partition --active
```

Comment:
same comment about 'K' as for omnect-os.phytec.wks.in above

Reply:
done